firefox-ios/Client/Frontend/TrackingProtection/CertificatesViewModel.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import X509
+
+// TODO FXIOS-9853 ⁃ Enhanced Tracking Protection Redux
+final class CertificatesViewModel {
+    let topLevelDomain: String
+    let title: String
+    let URL: String
+    var certificates = [Certificate]()
+    var selectedCertificateIndex: Int = 0
+
+    init(topLevelDomain: String, title: String, URL: String, certificates: [Certificate]) {
+        self.topLevelDomain = topLevelDomain
+        self.title = title
+        self.URL = URL
+        self.certificates = certificates
+    }
+
+    func getCertificateValues(from data: String) -> [String: String] {
```

Comment:
Sorry I missed this before. Maybe you can rename the input `from input: String` like you have with `getDNSNamesList`? 

It is a tiny bit confusing to call a string "data" since Swift also has a `Data` type.